libcodechecker/libhandlers/cmd.py

```diff
@@ -211,6 +228,20 @@ def __register_diff(parser):
                        help="Show results that appear in both the 'base' and "
                             "the 'new' run.")
 
+    def __handle(args):
```

Comment:
Could you rename this to something more descriptive?

Reply:
This is the usual name for this method in `server` and various other commands which have extra rules on their invocation.
This method only lives in the local scope of the registering function and isn't used anywhere else.